nautobot/cloud/navigation.py

```diff
@@ -46,6 +46,22 @@
                             ),
                         ),
                     ),
+                    NavMenuItem(
```

Comment:
Let's think carefully about ordering of the Cloud menu items. To match the other menus, should `Cloud Services` be located above `Cloud Types` at least?